examples/getting-started/display-static-map.ipynb

```diff
@@ -74,7 +72,7 @@
     "# Generate a URL that displays a global DEM\n",
     "url = ee.Image(\"USGS/SRTMGL1_003\").getThumbUrl({'min':0, 'max':3000})\n",
     "# Display the image in the notebook.\n",
-    "Image(url=url)"
+    "Image(url=url, embed=True, format='png')"
```

Comment:
This file appears unrelated to the PR. Could you move this to a separate PR, or expand the description of this PR if you think it is related?